config/routes.rb

```diff
@@ -0,0 +1,61 @@
+Rails.application.routes.draw do
+
+  resources :passengers do
+    resources :trips, only: [:new, :create, :index]
+  end
+
+  resources :trips, except: [:new, :create]
+
+  resources :drivers
+
+
+  get 'user_interfaces', to: 'user_interfaces#index', as: 'user_interfaces'
```

Comment:
You don't need this route here, as you're just using `user_interfacesController` as the root path.